### DISCUSSION TITLE
Are tooltips avaible in SQLPage ?

### DISCUSSION THREAD

**Discussion by ThierryM** [sqlpage/SQLPage#343] [Q&A] [answered]
Hi,
Is it possible to have tooltips over SQLPage elements ?
Regards.

**Comment by ThierryM** [comment]
I have an answer for text component with the Markdown syntax and adding for instance "Infobulle" after the link beetween () :
```
 select 
    'text' as component,
   '##### Légende :
![Cartable](./icons/briefcase.svg "Infobulle") Dossier de l''élève' as contents_md;
```
The result is: 
![Sélection_646](https://github.com/lovasoa/SQLpage/assets/1470067/cff6c333-c3aa-4e3b-8188-50e581e99374)

But is it possible for a button ?

**Comment by lovasoa** [comment]
That's not yet possible but it would be a nice addition indeed

**Comment by lovasoa** [comment, marked as answer]
I just added a `tooltip` property to the button component.

You can try it already with `lovasoa/sqlpage:main` on docker, or wait for the next stable release.
